aci/resource_aci_vzrssubjfiltatt.go

```diff
@@ -0,0 +1,272 @@
+				ValidateFunc: validation.StringInSlice([]string{
+					"deny",
+					"permit",
+				}, false),
+			},
+			"directives": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+					ValidateFunc: validation.StringInSlice([]string{
+						"log",
+						"no_stats",
+						"none",
+					}, false),
+				},
+			},
+			"priority_override": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					"default",
+					"level1",
+					"level2",
+					"level3",
+				}, false),
+			},
+			"filter_dn": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+		}),
+	}
+}
+
+func getRemoteSubjectFilter(client *client.Client, dn string) (*models.SubjectFilter, error) {
+	vzRsSubjFiltAttCont, err := client.Get(dn)
+	if err != nil {
+		return nil, err
+	}
+	vzRsSubjFiltAtt := models.SubjectFilterFromContainer(vzRsSubjFiltAttCont)
+	if vzRsSubjFiltAtt.DistinguishedName == "" {
+		return nil, fmt.Errorf("SubjectFilter %s not found", vzRsSubjFiltAtt.DistinguishedName)
+	}
+	return vzRsSubjFiltAtt, nil
+}
+
+func setSubjectFilterAttributes(vzRsSubjFiltAtt *models.SubjectFilter, d *schema.ResourceData) (*schema.ResourceData, error) {
+	dn := d.Id()
+	d.SetId(vzRsSubjFiltAtt.DistinguishedName)
+	if dn != vzRsSubjFiltAtt.DistinguishedName {
+		d.Set("contract_subject_dn", "")
+	}
+	vzRsSubjFiltAttMap, err := vzRsSubjFiltAtt.ToMap()
+	if err != nil {
+		return d, err
+	}
+	d.Set("action", vzRsSubjFiltAttMap["action"])
+	directivesGet := make([]string, 0, 1)
+	for _, val := range strings.Split(vzRsSubjFiltAttMap["directives"], ",") {
+		directivesGet = append(directivesGet, strings.Trim(val, " "))
+	}
+	sort.Strings(directivesGet)
+	if directivesIntr, ok := d.GetOk("directives"); ok {
+		directivesAct := make([]string, 0, 1)
+		for _, val := range directivesIntr.([]interface{}) {
+			directivesAct = append(directivesAct, val.(string))
+		}
+		sort.Strings(directivesAct)
+		if reflect.DeepEqual(directivesAct, directivesGet) {
+			d.Set("directives", d.Get("directives").([]interface{}))
+		} else {
+			d.Set("directives", directivesGet)
+		}
+	} else {
+		d.Set("directives", directivesGet)
+	}
+	d.Set("priority_override", vzRsSubjFiltAttMap["priorityOverride"])
+	tnVzFilterName := GetMOName(d.Get("filter_dn").(string))
+	if tnVzFilterName != vzRsSubjFiltAttMap["tnVzFilterName"] {
+		d.Set("filter_dn", "")
+	}
+	d.Set("tn_vz_filter_name", vzRsSubjFiltAttMap["tnVzFilterName"])
+	return d, nil
+}
+
+func resourceAciSubjectFilterImport(d *schema.ResourceData, m interface{}) ([]*schema.ResourceData, error) {
+	log.Printf("[DEBUG] %s: Beginning Import", d.Id())
+	aciClient := m.(*client.Client)
+	dn := d.Id()
+	vzRsSubjFiltAtt, err := getRemoteSubjectFilter(aciClient, dn)
+	if err != nil {
+		return nil, err
+	}
+	schemaFilled, err := setSubjectFilterAttributes(vzRsSubjFiltAtt, d)
```

Comment:
setSubjectFilterAttributes uses d.GetOk or d.Get which are not available in import how is it still properly imported?